dev-libs/granite/granite-5.2.4.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+VALA_MIN_API_VERSION=0.40
+
+inherit xdg-utils meson vala
+
+DESCRIPTION="Elementary OS library that extends GTK+"
+HOMEPAGE="https://github.com/elementary/granite"
+SRC_URI="https://github.com/elementary/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3"
```

Comment:
This license is wrong ;)

Reply:
Why? Their repository has LGPL as the license: https://github.com/elementary/granite/blob/master/COPYING

Reply:
Check the source files and https://devmanual.gentoo.org/general-concepts/licenses/index.html especially the big warning part ;)

Reply:
Ah, is it that it should be `LGPL-3+`?

Reply:
That's it, didn't see more copyright lines, but you're always welcome to grep the source tree a bit ;)